test/integration/main_test.go

```diff
@@ -61,6 +61,12 @@ var (
 	gcsClient *storage.Client
 )
 
+type QueryResult struct {
```

Comment:
Why the query result looks like this? It looks weird as it doesn't match the bq schema we use.

Reply:
+1, it will be easier to understand this way. we can compare if each of them are what we expected separately.

Reply:
Changed to only `Data` and `Attributes`